README.md

```diff
@@ -34,71 +34,85 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. If you were to describe semantic HTML to the next cohort of students, what would you say?
 
-2. Name two big differences between ```display: block;``` and ```display: inline;```.
+- Semantic HTML is the concept of using more descriptive tags while marking up your html document. There are tags that do the same things but certain tags have a more descriptive name. This is useful for when you are working with other teammates who also have to read what you have wrote and may even be useful to yourself. The difference between a semantic tag and a non-semantic tag is the difference between a cardboard box and a box with a label. Both function the same but you will have an idea of what is in the labeled box before you even look in it because of the label.
+
+2. Name two big differences between `display: block;` and `display: inline;`.
+
+- The first big difference is how they interact with the elements around them. A block level element will push other elements to the next line on the page and puts itself on a new line on the page when added in.
+- The second big difference is you cannot set a width and height to an element that is inline.
 
 3. What are the 4 areas of the box model?
 
-4. While using flexbox, what axis does the following property work on: ```align-items: center```?
+The 4 areas of the box model from the outside in is:
+
+- 1. margin
+- 2. border
+- 3. padding
+- 4. content
+
+4. While using flexbox, what axis does the following property work on: `align-items: center`?
+
+- align-items: center; works on the cross axis. justify-content works on the main axis
```

Comment:
Great job recognizing the axis in the flexbox model, you will get to build cool projects with this as you go indepth into the curriculum